indico/modules/rb/client/js/modules/calendar/CalendarListView.jsx

```diff
@@ -101,26 +110,58 @@ class CalendarListView extends React.Component {
           {moment(day, 'YYYY-MM-DD').format('dddd, LL')}
         </Header>
         <Card.Group itemsPerRow={4} stackable styleName="day-cards-container">
-          {bookings.map(this.renderBooking)}
+          {bookings.map(booking => this.renderBooking(booking, day, linkData))}
         </Card.Group>
         <Divider hidden />
       </div>
     );
   };
 
-  renderBooking = booking => {
+  renderLink = (booking, day, linkData, linkBookingOccurrence) => {
+    if (linkData) {
+      const {reservation} = booking;
+      const boundaries = [moment(linkData.startDt), moment(linkData.endDt)];
+      const datesMatch =
+        moment(booking.startDt).isBetween(...boundaries, undefined, '[]') &&
+        moment(booking.endDt).isBetween(...boundaries, undefined, '[]');
+      const linkBtn = (
+        <Button
+          icon={<Icon name="linkify" />}
+          primary
+          disabled={!!booking.link || !datesMatch}
+          size="small"
+          circular
+          onClick={() =>
+            linkBookingOccurrence(reservation.id, day, linkData.id, () =>
+              this.refetchActiveBookings(false)
+            )
+          }
+        />
+      );
+      return (
+        <div style={{position: 'absolute', top: '5px', right: '5px'}}>
+          <Popup trigger={linkBtn} position="bottom center">
+            <Translate>
+              Link to <Param name="bookedFor" value={linkData.title} />
+            </Translate>
+          </Popup>
+        </div>
+      );
+    }
+  };
+
+  renderBooking = (booking, day, linkData) => {
     const {
-      actions: {openBookingDetails},
+      actions: {openBookingDetails, linkBookingOccurrence},
     } = this.props;
     const {reservation} = booking;
     const {room, isAccepted} = reservation;
     const key = `${reservation.id}-${booking.startDt}-${booking.endDt}`;
     const startTime = moment(booking.startDt, 'YYYY-MM-DD HH:mm').format('LT');
     const endTime = moment(booking.endDt, 'YYYY-MM-DD HH:mm').format('LT');
-
     return (
-      <Card styleName="booking-card" key={key} onClick={() => openBookingDetails(reservation.id)}>
-        <Card.Content>
+      <Card styleName="booking-card" key={key}>
+        <Card.Content onClick={() => openBookingDetails(reservation.id)}>
```

Comment:
This for some reason breaks the hover animation for the card (haven't looked into why). Maybe you could keep the `onClick` on the card and use `stopPropagation`?

Reply:
Yep, good suggestion. Thanks!